kubernetes/server/create.sh

```diff
@@ -1,24 +1,20 @@
 #!/bin/bash
 
 function kubectl_create() {
-  kubectl create -f scdf.yml
+  kubectl create -f scdf.yml --namespace $KUBERNETES_NAMESPACE
   READY_FOR_TESTS=1
   for i in $( seq 1 "${RETRIES}" ); do
-    SERVER_URI=$(kubectl get svc scdf | grep scdf | awk '{print $3}')
+    SERVER_URI=$(kubectl get svc scdf --namespace $KUBERNETES_NAMESPACE | grep scdf | awk '{print $3}')
     [ '<pending>' != $SERVER_URI ] && READY_FOR_TESTS=0 && break
```

Comment:
Instead of checking for `pending`, I guess we need to check for `80/TCP` as that will be the result for `awk '{print $3}')` until the external IP is assigned